h2/src/main/org/h2/table/RangeTable.java

```diff
@@ -144,7 +144,7 @@ public long getRowCount(Session session) {
 
     @Override
     public TableType getTableType() {
-        throw DbException.throwInternalError(toString());
+    	return TableType.SYSTEM_TABLE;
```

Comment:
This unimplemented error cropped up while testing recursive and non recursive paths - when the range function used as the internal way to manifest a  literal (select 1) value.